Sources/NavigationStack/NavigationStack.swift

```diff
@@ -48,11 +48,11 @@ public class NavigationStack: ObservableObject {
     fileprivate private(set) var navigationType = NavigationType.push
     /// Customizable easing to apply in pop and push transitions
     private let easing: Animation
-    
-    init(easing: Animation) {
+
+    public init(easing: Animation = .easeOut(duration: 0.2)) {
```

Comment:
Please, extract the default animation because it's used both by the NavigationStack and the NavigationStackView.